docs/source/interaction_model.rst

```diff
@@ -1,95 +1,75 @@
 Interaction model
-================
+================= 
 
-The interaction model defines the user-agent interactions that can take place in a conversation. We define it in terms a set of *user intents*, a set of *agent intents*, and *expected agent responses* to each user intent.
-Below, we specify the YAML format that is used for defining an interaction model and then discuss a specific interaction model (CIR6) that is shipped with the toolkit.
+The interaction model has two main responsibilities: (1) defining the allowed transitions between dialogue acts based on their intents and (2) updating the agenda based on a predefined dialogue strategy.
```

Comment:
A slightly extended version:
The interaction model is based on [(Zhang & Balog, 2020)](https://arxiv.org/abs/2006.08732) and defines the allowed transitions between dialogue acts based on their intents. In our implementation, the interaction model is also responsible for the updating of the agenda based on a predefined dialogue strategy.

Reply:
Done.